nvflare/app_common/aggregators/dxo_aggregator.py

```diff
@@ -61,7 +61,7 @@ def __init__(
 
         if name_postfix:
             self._name += name_postfix
-            self.logger = logging.getLogger(self._name)
+            self.logger = get_module_logger(self.__module__, f"{self.__class__.__qualname__}{name_postfix}")
```

Comment:
Do we want postfix here? or just use the "_name" if that's what we want?
Another question why do we need this "postfix" @holgerroth @ZiyueXu77 do you remember?

Reply:
The postfix is currently used to differentiate DXOAggregrators in InTimeAccumulateWeightedAggregator when processing a DXO of `DataKind.COLLECTION`